test/action.js

```diff
@@ -283,6 +293,20 @@ describe('Action', function() {
 					expect(siren.getFieldsByClass(/foo/)).to.be.an.instanceof(Array).and.to.be.empty;
 				});
 
+				it('getFieldByClasses', function() {
+					expect(siren.getFieldByClasses(['bonk', 'bork'])).to.have.property('name', 'foo3');
+					expect(siren.getFieldByClasses([/bonk/, /bork/])).to.have.property('name', 'foo3');
```

Comment:
Probably a somewhat-silly test case, but combining regexes and strings doesn't appear to be covered

Reply:
Good call, will do.